backend/app/api/endpoints/runtime_work.py

```
@@ -0,0 +1,188 @@
+        request=request,
+    )
+
+
+@router.post(
+    "/im-sessions",
+    response_model=BindRuntimeTaskIMSessionsResponse,
+    response_model_by_alias=True,
+)
+async def bind_runtime_task_im_sessions_endpoint(
+    request: BindRuntimeTaskIMSessionsRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)